WordPress/src/main/java/org/wordpress/android/viewmodel/activitylog/ActivityLogDetailViewModel.kt

```diff
@@ -95,8 +89,19 @@ class ActivityLogDetailViewModel
         rewindStatusService.stop()
     }
 
-    private fun onRewindClicked() {
-        _item.value?.let { _showRewindDialog.postValue(it) }
+    fun onRangeClicked(activity: FragmentActivity, range: FormattableRange) {
+        formattableContentClickHandler.onClick(activity, range)
```

Comment:
Have you considered moving this into the Activity, so we don't have an Android dependency with a lifecycle in the VM?

Reply:
Good point 👍